PowerPointLabs/PowerPointLabs/ThisAddIn.cs

```diff
@@ -485,6 +486,15 @@ private void ShutDownColorPane()
             }
         }
 
+        private void ShutDownPictureSlidesLab()
+        {
+            PictureSlidesLab.Views.PictureSlidesLabWindow pictureSlidesLabWindow = Globals.ThisAddIn.Ribbon.PictureSlidesLabWindow;

```

Comment:
This script is already `ThisAddIn` itself right? Maybe you could simplify this by changing it to just `Ribbon.PictureSlidesLabWindow` ?

Reply:
Done!